docs/pbs/index.md

```diff
@@ -1,111 +1,174 @@
+
+| Directive | Function     |
+| --------- | ------------ |
+| `#PBS -n hello_pbs` | Provides a job name. This name will be displayed by the scheduler for diagnostic and file output. If omitted, and a script is used to submit the job, the job's name is the name of the script. |
+| `#PBS -A <project_code>` | Specify a NCAR Project Accounting code resource allocation will be applicable to this job. (You will want to replace `<project_code>` with your project's specific code.) |
+| `#PBS -j oe` | Requests we combine any standard text output (o) and error (e) into one output file. (By default, PBS will write program output and error to different log files. This behavior is contrary to what many users expect from terminal interaction, where output and error are generally interspersed. This optional flag changes that behavior.) |
+| `#PBS -q main` | Specifies the desired PBS queue for this job |
+| `#PBS-l walltime=00:05:00` | Requests 5 minutes as the maximum job execution (walltime) time. Specified in HH:MM:SS format. |
+| `#PBS -l select=2:ncpus=128:mpiprocs=128` | Request a computational resource chunk, detailing the quantity and configuration of compute nodes required for this job. This example requests a selection of 2 nodes, where each node must have 128 CPU cores, each of which we will use as an MPI rank in our application. |
+
+Some directives like queues may vary from system to system. Please check out our [queues and charging](./charging.md) page for more informatiom
+
+##### Environment Setup
+Once your PBS directives are in place, there’s no restriction on what else is needed in script. This being said, CISL recommends users configure their environments to be tailored to what their job needs are at the start of the script. This usually includes explicit module setup, setting a `TMPDIR`, and setting any additional variables you might need within the script. 
+
+**Loading and reporting the specific module environment required for this job.**
+
+Defining the module environment is not required but highly recommended as a best practice to facilitate debugging or reproducing later.  If the module environment is not defined in the script then the job with run with the default module stack.  The default module stack is changed periodically which could lead to errors if you compiled an application with an older, incompatible default module environment.  Additional details on setting up your module environment can be found in the [modules](../environment-and-software/user-environment/modules.md) section.
+
+**Explicitly setting the `TMPDIR` variable.**
+
+As described here, many programs write temporary data to `TMPDIR`, which is usually small and shared amongst users. Specifying your own directory for temporary files can help you avoid the risk of your own programs and other users' programs failing when no more space is available.
+
+(Optional) Defining any environment variables specific to the chosen module environment.
```

Comment:
This is not demonstrated by the example and to me, is vague. I suggest it is cut.